skins/wmde19/templates/Base_Layout.html.twig

```diff
@@ -28,7 +28,9 @@
 </head>
 <body>
 {% block main %}
-    <div id="app" data-application-vars="{$ _context|json_encode|e('html_attr') $}" data-application-messages="{$ translations()|e('html_attr') $}" >
+    <div id="app" data-application-vars="{$ _context|json_encode|e('html_attr') $}" 
+    data-application-messages="{$ translations()|e('html_attr') $}" 
+    data-assets-path="{% if assets_path %}{$ assets_path $}{% else %}{$ basepath $}/skins/wmde19/images{% endif %}">
```

Comment:
This does not work in compiled mode (`assets_path`empty), because the components add `/images/imagename.svg` instead of `/imagename.svg`.